### PR TITLE
[substrate] remove pullSecret from charts

### PR DESCRIPTION
**Pull request by TonyRowntree**
Signed-off-by: TonyRowntree <33454202+TonyRowntree@users.noreply.github.com>

**Changelog**
- Removed pullSecret key from charts

tested on uat-network

 
 

**Linked issue**
#INFRA-139

### CODE REVIEW

File: examples/dscp-app/charts/dscp-api/templates/_helpers.tpl [hunk 1]
```diff
@@ -54,15 +54,3 @@ app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 {{- end }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
 {{- end }}
-
```
Comment (sownak) [inline examples/dscp-app/charts/dscp-api/templates/_helpers.tpl]: This part is fine, no need to remove

File: examples/dscp-app/charts/inteli-api/values.yaml [hunk 2]
```diff
@@ -42,7 +42,6 @@ image:
   repository: ghcr.io/digicatapult/inteli-api
   pullPolicy: IfNotPresent
   tag: 'v1.28.0'
-  pullSecrets: ['ghcr-digicatapult']
```
Comment (sownak) [inline examples/dscp-app/charts/inteli-api/values.yaml:45]: only remove the value here

File: platforms/substrate/charts/substrate-node/values.yaml [hunk 3]
```diff
@@ -28,7 +28,6 @@ googleCloudSdk:
     tag: slim # more lightweight than the full image and still contains gsutil
   #serviceAccountKey: ""
 
-imagePullSecrets: []
```
Comment (sownak) [inline platforms/substrate/charts/substrate-node/values.yaml]: this need not be removed

**Review by sownak — CHANGES_REQUESTED** [review]

**Review by sownak — APPROVED** [review]

**Comment by dblane-digicatapult** [comment]
Hi,

I think if you are removing the pullSecrets then you should switch to using the containers on dockerhub. As far as I know ghcr still requires auth.

https://hub.docker.com/u/digicatapult

**Pull request merged**